plugins/apps/threatbus_cif3/plugin.py

```diff
@@ -0,0 +1,98 @@
+from queue import Queue
+import threading
+from cifsdk.client.http import HTTP as Client
+
+from threatbus_cif3.message_mapping import map_to_cif
+import threatbus
+
+"""Threatbus - Open Source Threat Intelligence Platform - plugin for CIFv3"""
+
+plugin_name = "cif3"
+cif = None
```

Comment:
```suggestion
```
For the current implementation, I don't think it is required to have `cif` as a global variable. It is initiated once and then only used in the `receive_intel` function. Since there are no other users, it does not make sense to have it global. That also makes the `lock` obsolete.